### PR TITLE
GLT-553 fixes for Sample builder and Dto

### PR DESCRIPTION
**Pull request by callunity**
- fixes issues identified in PR #177

### CODE REVIEW

File: core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractSample.java [hunk 1]
```diff
@@ -751,10 +751,11 @@ public Sample build() {
           return SampleImpl.sampleAnalyte(this);
         } else if (sampleTissue != null) {
           return SampleImpl.sampleTissue(this);
-        } else {
+        } else if (sampleAdditionalInfo.getSampleClass().getSampleCategory() == "Tissue Processing"){
```
Comment (apmasell) [inline core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractSample.java]: This needs to be `.equals`

Comment (callunity) [inline core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractSample.java]: Riiight. Java.

**Comment by tonydebat** [comment]
:+1:

**Comment by apmasell** [comment]
:+1:

**Pull request merged**